src/iGameGUI.c

```diff
@@ -1035,6 +1036,7 @@ struct ObjApp * CreateApp(void)
 		MUIA_Application_Copyright, GetMBString(MSG_AppCopyright),
 		MUIA_Application_Description, GetMBString(MSG_AppDescription),
 		MUIA_Application_HelpFile, "iGame.guide",
+		MUIA_Application_DiskObject, appIcon=GetDiskObject("PROGDIR:iGame"),
```

Comment:
Why does this variable exist when all that happens is that it gets assigned to and nothing else?

Reply:
You were right. I removed it. Thanks for your comment.